toplevel/vernac.ml

```diff
@@ -145,6 +152,7 @@ let rec interp_vernac sid po (loc,com) =
       let com = if !Flags.time then VernacTime (loc,com) else com in
       interp com
     with reraise ->
+      ignore(Stm.edit_at sid);
```

Comment:
In a sense, to easy the life of clients I wanted the function to have the invariant of leaving the stm always in a consistent state.